### PR TITLE
Multi-tenancy support for GQL Get{} queries

### PR DESCRIPTION
**Pull request by parkerduckworth**
### What's being changed:

- Support for GQL Get{} queries of multi-tenancy-enabled classes

### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: test/acceptance/multi_tenancy/gql_get_tenant_objects_test.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+		})
+	})
+
+	t.Run("GQL Get tenant objects", func(t *testing.T) {
+		expectedIDs := map[strfmt.UUID]bool{}
+		query := fmt.Sprintf(`{Get{%s(tenantKey:%q){_additional{id}}}}`, testClass.Class, tenantName)
+		result := graphqlhelper.AssertGraphQL(t, helper.RootAuth, query)
+		for _, obj := range result.Get("Get", testClass.Class).AsSlice() {
+			id := obj.(map[string]any)["_additional"].(map[string]any)["id"].(string)
+			expectedIDs[strfmt.UUID(id)] = true
```
Comment (aliszka) [inline test/acceptance/multi_tenancy/gql_get_tenant_objects_test.go:107]: I don't see where this map is inited with expected uuids. Seems like only found ones are added here. So test will not fail even if map is empty

File: test/acceptance/multi_tenancy/gql_get_tenant_objects_test.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+		}
+		for id, found := range expectedIDs {
+			if !found {
+				t.Fatalf("expected to find id %q, but didn't", id)
+			}
+		}
+	})
+}
+
+func TestGQLGetTenantObjects_MissingTenantKey(t *testing.T) {
```
Comment (aliszka) [inline test/acceptance/multi_tenancy/gql_get_tenant_objects_test.go:117]: can we add some objects and verify results does not contain any object, only error?

File: test/acceptance/multi_tenancy/gql_get_tenant_objects_test.go (hunk 1 above)
Comment (parkerduckworth) [reply test/acceptance/multi_tenancy/gql_get_tenant_objects_test.go]: Nice catch, fixed

File: test/acceptance/multi_tenancy/gql_get_tenant_objects_test.go (hunk 2 above)
Comment (parkerduckworth) [reply test/acceptance/multi_tenancy/gql_get_tenant_objects_test.go:126]: Nice catch, fixed

**Review by aliszka — APPROVED** [review]

**Pull request merged**